scripts/build.mjs

```diff
@@ -78,10 +55,12 @@ async function main() {
       {
         ...pkg,
         files: ["dist-*/**", "bin/**"],
-        main: "dist-node/index.js",
-        module: "dist-web/index.js",
-        types: "dist-types/index.d.ts",
```

Comment:
Can you leave this field in. TypeScript sometimes has trouble, and it doesn't display on npm